<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHEMETOV</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>535</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>График</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>499</y>
     <width>71</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчёт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>499</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчёт</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>11</y>
     <width>1101</width>
     <height>481</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>№ Интервала</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Границы 
интервала</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Середина 
интервала</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Число 
отказов</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Относительная 
частота</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Эмпирич. 
надёжность</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Эмпирич. 
интенсивн. 
отказов</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сер. инт. 
*
инт. отказа</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Вер. безотк. 
работы на инт.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Вер. отказа 
на инт.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Плотность 
распределения</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="9" column="0">
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>519</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во агрегатов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>500</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Макс. кол-во часов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>538</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во неиспр агр.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
